packages/tests/src/teardown.ts

```diff
@@ -26,7 +29,7 @@ export async function tearDownNodes(
     }
   });
 
-  const stopWakuNodes = wakuNodes.map(async (waku) => {
```

Comment:
I recently thought about this and `remoteNode` might be the best descriptive

Reply:
This code is outdated and not sure I understand you suggestion.

Reply:
I think I saw a `wNode` name and we have a weird "NimGoWaku" naming var in the code. I am saying that maybe we should use "remoteNode" as a terminology instead. It's a nitpick. Cc @waku-org/js-waku-developers

Reply:
oh, ok that makes sense. Will update it with a next PR